### ISSUE TITLE
Conflict between PagedList and X.PagedList

### ISSUE THREAD

**Issue by ricardovsilva** [ernado-x/X.PagedList#12]
When you install the nugget package for X.PagedList.Mvc, it also installs PagedList. The problem is that IPagedList exists in both dlls. And that makes visual studio halt error and not compile.

**Comment by ricardovsilva** [comment]
After delete PagedList.dll from references, I'm getting execution time error related to temporary files. I already deleted these temporaries but the problem persists.

```The type 'PagedList.IPagedList' exists in both 'c:\Users\xxx\AppData\Local\Temp\Temporary ASP.NET Files\root\b79c1da8\6c072ce6\assembly\dl3\a5d9c6c5\d6d07659_3510d101_0\PagedList.DLL' and 'c:\Users\xxx\AppData\Local\Temp\Temporary ASP.NET Files\root\b79c1da8\6c072ce6\assembly\dl3\1556ef00\b6769801_7f11d101\X.PagedList.DLL'```

**Comment by ernado-x** [comment]
Plaease view updated package. I install it on new project and have no conflicts.

https://www.nuget.org/packages/X.PagedList.Mvc/

**Comment by MNeverOff** [comment]
Simply rolling new build may not help this gentleman.

I've encountered the same issue and had to do next steps:
1) Go to view to the namespace declarations 
(@using PagedList.Mvc;
@using PagedList;)
2) RMD on namespace -> Go to definition,

From there you'll find the location of the PagedList .dll, which was left over by the VS for some god forsaken reason.
I deleted the DLL and everything went back to normal.
